src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -556,11 +556,15 @@ CreateDistributedTable(Oid relationId, Var *distributionColumn, char distributio
 	{
 		List *partitionList = PartitionList(relationId);
 		Oid partitionRelationId = InvalidOid;
+		char *parentRelationName, *nspname;
+		nspname = get_namespace_name(get_rel_namespace(relationId));
+		parentRelationName = quote_qualified_identifier(nspname, get_rel_name(
```

Comment:
We have to bend over backwards to make the name fully qualified, if not for SQL
SELECT create_distributed_table('ppp', 'measureid', colocate_with:='soo.moo');
'moo' may be mapped falsely to another table in the current search_path